engine/src/main/java/com/sandboni/core/engine/Application.java

```diff
@@ -45,7 +45,7 @@ Arguments buildArguments() {
                 .toChangeId(getValue(SystemProperties.TO))
                 .repository(getValue(SystemProperties.REPOSITORY))
                 .filter(getValue(SystemProperties.FILTER))
-                .runSelectiveMode(Boolean.parseBoolean(getValue(SystemProperties.SELECTIVE_MODE)))
+                .runOnConfigChange(Boolean.parseBoolean(getValue(SystemProperties.SELECTIVE_MODE)))
```

Comment:
We have changed a lot of things in the API. I think that we need to be more cautious with the artifact versioning. I suggest to bump up the version to 1.2.0

Reply:
Agreed